homeassistant/components/remote/__init__.py

```diff
@@ -0,0 +1,155 @@
+"""
+Component to interface with universal remote control devices.
+
+For more details about this component, please refer to the documentation
+at https://home-assistant.io/components/remote/
+"""
+from datetime import timedelta
+import logging
+import os
+
+import voluptuous as vol
+from homeassistant.config import load_yaml_config_file
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.entity import ToggleEntity
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    STATE_ON, SERVICE_TURN_ON, SERVICE_TURN_OFF, ATTR_ENTITY_ID)
+from homeassistant.components import group
+from homeassistant.helpers.config_validation import PLATFORM_SCHEMA  # noqa
+
+ATTR_DEVICE = 'device'
+ATTR_COMMAND = 'command'
+ATTR_ACTIVITY = 'activity'
+SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SYNC = 'sync'
+ATTR_DEFAULT = ''
+
+DOMAIN = 'remote'
+SCAN_INTERVAL = 30
+
+GROUP_NAME_ALL_REMOTES = 'all remotes'
```

Comment:
I think there's a valid use case for the all_remotes group. I have my instance set to check if all are off when I leave to make sure i didn't forget to turn a TV off